src/components/Overview.jsx

```diff
@@ -131,60 +182,50 @@ class Overview extends React.Component {
     }
   }
 
-  getSizes() {
-    let sizes = [];
-    for (var key in this.state.currentStyle.skus) {
-      sizes.push(this.state.currentStyle.skus[key].size)
-    }
-    this.setState({ 'sizes': sizes });
-  }
-
   pickSize(n) {
+    n.preventDefault();
     let quantities = [];
     for (var key in this.state.currentStyle.skus) {
       if (this.state.currentStyle.skus[key].size === n.target.value) {
-        for (let i = 0; i <= this.state.currentStyle.skus[key].quantity; i++) {
+        for (let i = 1; i <= this.state.currentStyle.skus[key].quantity; i++) {
           if (i <= 15) {
             quantities.push(i);
           }
         }
       }
     }
-    this.setState({ 'quantities': quantities })
+    this.setState({ 'currentSize': n.target.value, 'quantities': quantities })
+  }
+
+  pickQuantity(n) {
+    n.preventDefault();
+    this.setState({ 'currentQuant': n.target.value })
   }
 
   styleSelect(style) {
-    this.setState({ 'currentStyle': style })
+    this.setState({ 'currentStyle': style, 'photoIndex': 1 })
+  }
+
+  movePhoto(n) {
+    let destination = this.state.photoIndex += n;
+    if (destination >= 1 && destination <= this.state.currentStyle.photos.length) {
+      this.setState({ 'photoIndex': destination })
+    }
   }
 
   render() {
       return (
       <div>
         <div className='overview-container'>
           <div className="gallery-container">
-            <Gallery style={this.state.currentStyle} />
+            <Gallery style={this.state.currentStyle} index={this.state.photoIndex} movePhoto={this.movePhoto.bind(this)}/>
           </div>
           <section className="product-info">
             <ProductInfo product={this.props.product} style={this.state.currentStyle} onSale={this.state.onSale}/>
-            <div>
             <StyleSelector style={this.state.currentStyle.name} styles={this.state.productStyles} onClick={this.styleSelect.bind(this)}/>
-          </div>
```

Comment:
Great use of replacing chunks of code with a component that does the same thing. Makes the reviewing process very easy to read your code.